disk/datadog_checks/disk/disk.py

```diff
@@ -65,6 +65,22 @@ def __init__(self, name, init_config, instances):
         self._compile_tag_re()
         self._blkid_label_re = re.compile('LABEL=\"(.*?)\"', re.I)
 
+        deprecations = {
+            'file_system_whitelist': 'file_system_include',
+            'file_system_blacklist': 'file_system_exclude',
+            'device_whitelist': 'device_include',
+            'device_blacklist': 'device_exclude',
+            'mount_point_whitelist': 'mount_point_include',
+            'mount_point_blacklist': 'mount_point_exclude',
+            'excluded_filesystems': 'file_system_exclude',
+            'excluded_disks': 'device_exclude',
+            'excluded_disk_re': 'device_exclude',
+            'excluded_mountpoint_re': 'mountpoint_exclude',
```

Comment:
I think this should be
```suggestion
            'excluded_mountpoint_re': 'mount_point_exclude',
```